src/components/FirstCommentWelcome/FirstCommentWelcome.tsx

```diff
@@ -1,42 +1,166 @@
+  outline: none;
+  width: 400px;
+  border-radius: 18px;
+  padding: 5px 15px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+  :focus {
+    border-color: ${neutral[46]};
+  }
+`;
+
+const textStyling = css`
+  ${textSans.xsmall()};
+`;
+
+const cancelButtonStyles = css`
+  margin-left: 20px;
+`;
+
+const errorTextStyles = css`
+  margin: 0;
+  ${textSans.xsmall({ fontWeight: "bold" })};
+  color: red;
+`;
+
+const imputWrapperStyles = css`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const FirstCommentWelcome = ({
+  body,
+  error = "",
+  submitForm,
+  cancelSubmit
+}: {
+  body: string;
+  error?: string;
+  submitForm: (userName: string) => void;
+  cancelSubmit: () => void;
+}) => {
+  const [previewBody, setPreviewBody] = useState<string>("");
+  const [userName, setUserName] = useState<string>("");
+
+  useEffect(() => {
+    const fetchShowPreview = async () => {
+      try {
+        const response = await preview(body);
+        setPreviewBody(response);
+      } catch (e) {
+        // TODO: handle errors?
+        setPreviewBody("");
+      }
+    };
+    fetchShowPreview();
+  });
 
-export const FirstCommentWelcome = () => {
   return (
-    <>
+    <form
+      onSubmit={e => {
+        e.preventDefault();
+        submitForm(userName);
+      }}
+    >
       <div>
         <h3
           className={css`
-            ${textSans.small()};
-            font-weight: bold;
+            ${textSans.medium({ fontWeight: "bold" })};
           `}
         >
-          Welcome William Lacroix, you’re about to make your first comment!
+          Welcome, you’re about to make your first comment!
         </h3>
       </div>
       <div>
-        <p>
-          Welcome William Lacroix, you’re about to make your first comment!
+        <p className={textStyling}>
           Before you post, we’d like to thank you for joining the debate - we’re
           glad you’ve chosen to participate and we value your opinions and
-          experiences. Please choose your username under which you would like
-          all your comments to show up. You can only set your username once.
+          experiences.
+        </p>
+        <p className={textStyling}>
+          Please choose your username under which you would like all your
+          comments to show up. You can only set your username once.
         </p>
       </div>
-      <div>
-        Username: Must be 6-20 characters, letters and/or numbers only, no
-        spaces.
-        <input />
+      <div className={cx(imputWrapperStyles, textStyling)}>
+        <label>
+          <span
+            className={css`
+              font-weight: bold;
+            `}
+          >
+            Username:{" "}
+          </span>
+          Must be 6-20 characters, letters and/or numbers only, no spaces.
+        </label>
+        <input
+          className={inputStyles}
+          type="text"
+          value={userName}
+          onChange={e => setUserName(e.target.value)}
+        />
```

Comment:
We should see about using the design system for this input

Reply:
Currently under discussion with design, will come back to this once we have design's input